nolabs/modules/biobuddy/data_models/message.py

```diff
@@ -27,6 +27,7 @@ class FunctionCall:  # Should be completely the same as in api models
 
 @dataclass
 class Message:  # Should be completely the same as in api models
```

Comment:
Why is this requirement?

Reply:
To easily parse/interpret Different types of messages from json to pydantic object